bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_cr-scc-hostmount-anyuid.yaml

```diff
@@ -6,12 +6,12 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
     rbac.authorization.kubernetes.io/autoupdate: "true"
-  name: system:openshift:scc:hostmount
```

Comment:
These names are effectively API, you cannot change them.

Reply:
I see, I thought there was a specific "scc-hostmount.yaml" and that `scc-hostmnount-anyuid.yaml` was different but now I see only the latter exists, I'll turn it back